server/controllers/articles.controller.js

```diff
@@ -24,4 +24,16 @@ exports.getComments = (req,res,next) => {
     }).catch((err)=>{
         next(err)
     })
+}
+
+exports.patchArticle = (req,res,next) => {
+    const {body} = req
+    const {article_id} = req.params
+    fetchArticle(article_id).then(()=>{
+        return updateArticle(article_id,body)
+    }).then((article)=>{
+        res.status(200).send({article})
+    }).catch((err) => {
+        next(err)
+    })
```

Comment:
We don't need to explicitly check that an article exists here - we can handle this functionality in the model -  if the article doesn't exist as it won't return anything and we can use a Promise.reject.